game/Bird.java

```diff
@@ -101,7 +101,7 @@ public void update() {
         if (dead) return;
 
         double birdYPossition = (double)yPos/Resources.HEIGHT;
-        double nextTubeHeigth = ((Resources.nextTube.getY() + Resources.TUBE_HEIGHT + Resources.TUBE_GAP_DISTANCE/2)-560+Resources.TUBE_HEIGHT)/(-260+560); // 80-380
+        double nextTubeHeigth = ((Resources.nextTube.getY() + Resources.TUBE_HEIGHT + Resources.TUBE_GAP_DISTANCE/2)-560+Resources.TUBE_HEIGHT)/(700); // 80-380
```

Comment:
I run the code several times for hours. I noticed that the birds could pass tubes when they were nearly at equal height but when there was a big difference then they accelerated to much or dropped too low. They learnet which way to fly but they overdone it. This change fixed this issue for me.